using_images/s2i_images/php.adoc

```diff
@@ -158,9 +165,18 @@ exec` command] to access the container.
 [[php-hot-deploy]]
 
 == Hot Deploying
-Hot deployment allows you to quickly make and deploy changes to your application
-without having to generate a new S2I build. Hot deployment works in this image
-out of the box.
+Hot deployment allows you to quickly make and deploy changes to your application without having to
+generate a new S2I build. In order to immediately pick up changes made in your application source
+code, you need to run your built image with the `OPCACHE_REVALIDATE_FREQ=0` environment variable.
+For example, see link:../../dev_guide/new_app.html#specifying-environment-variables[the `oc new-app`
+command]. You can use link:../../dev_guide/environment_variables.html#set-environment-variables[the
+`oc env` command] to update environment variables of existing objects.
```

Comment:
Re: command/link markup, I didn't love my solution. I've been convinced in https://github.com/openshift/openshift-docs/pull/1043/files#r41747221 to just keep the link to the backticked command itself.